.azure/templates/run-bvt.yml

```diff
@@ -44,6 +44,8 @@ jobs:
       filePath: scripts/prepare-machine.ps1
       ${{ if and(eq(parameters.kernel, true), eq(parameters.testCerts, true)) }}:
         arguments: -Configuration Test -TestCertificates -SignCode
+      ${{ if and(eq(parameters.kernel, false), eq(parameters.testCerts, true), eq(parameters.extraArtifactsDir, 'Xdp')) }}:
```

Comment:
These are not "if/else" type statements, so since the one right after will also be true, I expect you to get errors.

Reply:
I didn't get errors, but I do see in the log that it ended up with
'c:\a\w\1\s\scripts\prepare-machine.ps1' -Configuration Test -TestCertificates

so the third case must have overridden my new one.